### PR TITLE
OSDOCS-10920: adds 4.16.4 relnote MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.16

Issue:
[OSDOCS-10920](https://issues.redhat.com/browse/OSDOCS-10920)

Link to docs preview:
[4-16-4 release notes](https://79176--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes.html#microshift-4-16-4-dp_release-notes)

QE review:
- N/A stock text

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/label -peer-review-in-progress

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 1]
```diff
@@ -223,4 +223,13 @@ For the latest images included with {microshift-short}, view the contents of the
 [id="microshift-4-16-3-bug-fixes_{context}"]
 ==== Bug fixes
 
-* A mistake in validation allowed empty entries in the `listenAddress` configuration parameter. The empty entries prevented {microshift-short} from starting and rendered a difficult-to-understand error message. Empty entries in the list are now filtered.
\ No newline at end of file
+* A mistake in validation allowed empty entries in the `listenAddress` configuration parameter. The empty entries prevented {microshift-short} from starting and rendered a difficult-to-understand error message. Empty entries in the list are now filtered.
```
Comment (mletalie) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:226]:
```suggestion
* An error in validation allowed empty entries in the `listenAddress` configuration parameter. The empty entries prevented {microshift-short} from starting and rendered a difficult-to-understand error message. Empty entries in the list are now filtered.
```

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done
/lgtm

**Comment by mletalie** [comment]
Hello @ShaunaDiaz,
   Just one small suggestion, otherwise LGTM. Thanks!

File: microshift_release_notes/microshift-4-16-release-notes.adoc (hunk 1 above)
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-16-release-notes.adoc:226]: Huh, that's last week's release note. Interesting how it popped up in the diff. Fix it anyway! TY!

**Pull request merged**